### DISCUSSION TITLE
How to import large dumps initially?

### DISCUSSION THREAD

**Discussion by gopeter** [planetscale/discussion#149] [Q&A]
I'm trying to switch from AWS Aurora to Planetscale and have some large database that I need to migrate. So I've created a dump with `mysqldump` and tried to import it via CLI, but I always get EOF errors:

```
➜  ~ pscale shell my-db staging
my-db/staging> source staging_dump.sql
ERROR 1105 (HY000): rpc error: code = Unavailable desc = error reading from server: EOF
ERROR 1105 (HY000): rpc error: code = Unavailable desc = error reading from server: EOF
```

Is there a way to insert ~2 GB of SQL files to my new Planetscale DB?

**Comment by johnyerhot** [comment]
@gopeter hey! I'm curious if you still hit any snags using our [imports](https://docs.planetscale.com/concepts/database-imports) feature? It's specifically designed to bring over large datasets.

**Reply by gopeter** [reply]
I wanted to use this, but AWS Aurora does not allow external connections, so the importer would need an option for SSH tunnels 😬
